phys2bids/interfaces/txt.py

```diff
@@ -10,10 +10,64 @@
 import numpy as np
 
 from phys2bids.physio_obj import BlueprintInput
+from operator import itemgetter
+from collections import Counter
 
 LGR = logging.getLogger(__name__)
 
 
+def check_multifreq(timeseries, freq, start=0, leftout=0):
+    """
+    Checks if there are channels with different frequency than the maximum one
+
+    Parameters
+    ----------
+    timeseries: list
+        list with channels only in np array format
+    freq : list
+        list with the maximun frequency
+    start : integer
+        first sample of the channel to be considered
+    leftout : integer
+        number of sample at the end of the channel that are not considered
+        This is done  so this process doesn't take forever
+    Returns
+    -------
+    mfreq: list
+        new list with the actual frequency of the channels
+    """
+    mfreq = []
+    # for each channel check frequency
+    max_equal = 1
+    for idx, chann in enumerate(timeseries):
```

Comment:
A for loop with a while loop inside that at the same time has another for loop inside worries me. This could end up being problematic. We should probably think of a better way of handling this, probably in a future update.

Reply:
We can think about this in the future but this was the easiest way to implement it

Reply:
Drop an issue about it!

Reply:
As soon as we merge